config/sync/core.entity_view_display.eventinstance.default.card.yml

```diff
@@ -144,6 +161,7 @@ hidden:
   field_event_state: true
   field_event_title: true
   field_external_admin_link: true
+  field_screen_names: true
```

Comment:
Please consider what the entry `event_screen_name` is about.

As far as I can tell the new field is called `event_screen_names` or `field_screen_names`.

Reply:
Looks like cruft from earlier attempt. Tried removing them by hand and things still work. But I'm annoyed that Drupal will just let them be when `cim`/cex`ing.